core/domain/collection_domain.py

```diff
@@ -1102,6 +1105,20 @@ def delete_skill(self, skill_id):
 
         del self.skills[skill_id]
 
+    def get_acquired_skills_from_explorations(self, exploration_ids):
```

Comment:
Should be get_acquired_skill_ids_...? Similarly, please distinguish skill_ids from skills elsewhere.

Reply:
Updated.